pkg/setup/health_test.go

```diff
@@ -341,6 +341,7 @@ func TestHealthReporterRetry(t *testing.T) {
 			for _, st := range tc.Steps {
 				stepCh <- st
 				<-stateChange
+				time.Sleep(500 * time.Millisecond)
```

Comment:
in line 336, we send a message to `stateChange` after setting the report to healthy.
Why do we need to sleep here, when all the next check does is test if we area ready?